UICatalog/Scenarios/AutoSizeAndDirectionText.cs

```diff
@@ -7,18 +7,23 @@ public class AutoSizeAndDirectionText : Scenario {
 		public override void Setup ()
 		{
 			var text = "Hello World";
+			//var text = "Hello World 你";
```

Comment:
?

Reply:
I tested with wide runes and I commented to avoid rewrite all the time, but I'll automate that.